tests/test_backend.py

```diff
@@ -211,7 +211,7 @@ def test_set_call_empty_pipeline(self, cache: RedisCache, mocker: MockerFixture)
         cache.set(key, value, client=pipeline)
 
         if isinstance(cache.client, herd.HerdClient):
-            default_timeout = cache.client._backend.default_timeout
+            default_timeout = cache.client._backend.default_timeout or 0
```

Comment:
why this?

Reply:
According to mypy default_timeout can be None

Reply:
I mean, it's a test, which can also be not typed technically, but I'm not a fan of using `or` because I find it not readable, can the default be set in another line with an if or something?

Reply:
Ignoring mypy for that line would be an option with `# type: ignore`.

I totally agree with you that this makes no sense at all.

Reply:
I like the `ignore` solution

Reply:
Added the ignore, feels as the right thing to do over here.